# goal
uint8 ARM_RIGHT = 1
uint8 ARM_LEFT = 2
uint8 arm
---
# result
bool success
---
# feedback
uint8 APPROACH = 1
uint8 PRESSING = 2
uint8 CLOSING = 3
uint8 phase
